gensim/models/atmodel.py

```diff
@@ -19,15 +19,12 @@
 The model is closely related to Latent Dirichlet Allocation. The AuthorTopicModel class
 inherits the LdaModel class, and its usage is thus similar.
 
-Distributed computation and multiprocessing is not implemented at the moment, but may be
-coming in the future.
+The model was introduced by Rosen-Zvi and co-authors in 2004 and is described in [1]_
```

Comment:
Please don't use `[1]` references (this doesn't works properly with one of our sphinx plugin), intead of, you should use something like, here and everywhere
```
The model was introduced by Rosen-Zvi and co-authors in 2004 and is described in `The Author-Topic Model for Authors and Documents <https://arxiv.org/abs/1207.4169>`_
```

Reply:
Also, it's a good idea to add example of usage in current docstring (head of file)